src/orange/detail/BIHBuilder.cc

```diff
@@ -150,64 +149,67 @@ void BIHBuilder::construct_tree(VecIndices const& indices,
 
 //---------------------------------------------------------------------------//
 /*!
- * Seperate nodes into inner and leaf vectors and renumber accordingly.
+ * Partition nodes into inner and leaf vectors and renumber accordingly.
```

Comment:
The criteria is the type, isn't it? Or do you mean you're avoiding the term "partitioning" (a la [std::partition](https://en.cppreference.com/w/cpp/algorithm/partition)) because it's overloaded with the specific *spatial partitioning* done in BIH?

Reply:
Yes, one of the principle operations performed in this class is spatial partitioning, so it is best not to reuse the word "partition" to mean something totally different.

Reply:
Clarity is subjective, and we can agree to disagree, but FWIW:

Let's say in some MC code, you count the number of cross sections read from disk and print it out for diagnostic purposes. Calling this a "cross section tally" would be a grammatically appropriate use of the word "tally". However, "tally" has idiomatic significance within MC codes, so I would argue "cross section count" would be a less confusing choice.

That's what I am saying here: not that it isn't the same fundamental operation in each case, but rather that within a part of code pertaining to tree construction, for a reader who is not familiar with BIH construction (which will be most people, as BIH not as prevalent as BVH or k-d trees) I think seeing the word "partition" is going to make them think spatial partitioning is going on, when it's not.